### DISCUSSION TITLE
Using wxMaxima and Newton Raphson function to find minimum

### DISCUSSION THREAD

**Discussion by bella894** [wxMaxima-developers/wxmaxima#1591] [Q&A]
Ok, so I have calculated my stopping criterion based on 5 x 10^(-n) as the question asks for significant figures. So my epsilon, delta and eta values should be correct?! But it keeps telling me "The iterates seem to have moved outside the domain of definition of f. Stopping."

<img width="808" alt="Screenshot 2021-10-07 at 08 42 24" src="https://user-images.githubusercontent.com/91260617/136341732-21131c48-dc80-42e2-a105-0e6bf08ac0ec.png">

**Comment by daute** [comment]
Hi,
that seems to be not related to wxMaxima (the graphical user interface) but Maxima (the mathematical backend), so report the issue there: https://sourceforge.net/projects/maxima/

But I can't find a function `newton_raphson_minimize_1d()` **anywhere** in the current Maxima or wxMaxima sourcecode or documentation, is that a third party package you got from somewhere? In that case it should probably reported there. Or a function you wrote?

Best regards, Wolfgang

(And if the issue is 'text-only' (e.g. not a display problem somewhere in a graphical user interface), it is probably better to report the session as text - which can be copy-pasted - (too), not (only) as screenshot.
